api/src/org/apache/cloudstack/api/BaseCmd.java

```diff
@@ -192,6 +191,8 @@
     public AlertService _alertSvc;
     @Inject
     public UUIDManager _uuidMgr;
+    @Inject
+    public AnnotationService annotationService;
```

Comment:
this is not the usual pattern on API classes. I am adhering to the general practice in this case